.travis.yml

```diff
@@ -11,21 +12,40 @@ env:
 
 language: ruby
 
-rvm:
-  - 1.9
-  - 2.0
-  - 2.1
-  - 2.2
-  - 2.3
-  - 2.4
-  - jruby-9.1.10.0
-  - jruby-head
-  - ruby-head
-
 matrix:
   allow_failures:
     - rvm: jruby-head
     - rvm: ruby-head
   fast_finish: true
+  include:
+#    - rvm: 1.8.7-p371
+#      gemfile: gemfiles/ruby_1.8.gemfile
+    - rvm: jruby-1.7.27
+      gemfile: gemfiles/jruby_1.7.gemfile
+#    - rvm: 1.9.3-p551
```

Comment:
Do you think we should not try to support 1.9 & 2.0 ?

Reply:
No, they are only commented out for my local testing.  I will turn them back on once I get things stable.